Add support for WeTek Play 2 TV box

codesnake opened the pull request:
Add a project for WeTek Play 2 Amlogic S905 based TV box with DVB-S2/T/T2/C tuners (optional). Both eMMC and SD-card installations are supported.

lrusak commented:
Looks pretty good, can we move any of the Linux patches to our linux-amlogic tree?

codesnake commented:
@lrusak DTS patch will be moved for sure. Possibly DVB patches too.

chewitt commented on projects/WeTek_Play_2/install/updater-script:
```diff
@@ -0,0 +1,36 @@
+
+set_bootloader_env("upgrade_step", "1");
+show_progress(0.100000, 0);
+ui_print("OpenELEC Installed Successfully");
```
^^ freudian slip :)

chewitt replied:
You can fix that at the same time then!

chewitt commented on projects/WeTek_Play_2/options:
```diff
@@ -0,0 +1,165 @@
+    LTO_SUPPORT="yes"
+
+  # GOLD (Google Linker) support
+    GOLD_SUPPORT="yes"
```
some items here are not required after https://github.com/LibreELEC/LibreELEC.tv/commit/e2dd1e7bd100a0477f0499098c6db98ef5404901

lrusak commented:
Please rebase and bump amlogic kernel version when you get a chance :smile:

codesnake commented:
Rebased the PR: updated Amlogic kernel to 3.14-eb67324, fixed issues pointed by @chewitt, also added serial-console.service, same as recently added by @kszaq to WeTek Hub, WeTe Core and WeTek Play projects.

stefansaraev commented:
@codesnake fyi. to make refreshrate switch work properly while playing 4k material on 4k display I had to replace your 0001-aml-Fix-video-rect-calculation-when-using-GUI-scalin.patch with https://github.com/stefansaraev/xbmc/commit/77008ead0e4319517adc942b13019d2d74f79646 (part of #772)

codesnake commented:
Rebased once again: replaced video scaling patch with @wrxtasy's patch (thanks @stefansaraev for pointing out), updated header of platform_init script.

lrusak commented:
thanks! :+1:
